ui-modules/utils/server-status/server-status.js

```diff
@@ -73,7 +73,7 @@ export function BrServerStatusDirective() {
                         // we're now getting a new server error, possibly because the old error has expired
                         // but changing the message for the user would be confusing so don't do that!
                         // eg we get a 405 after a 307 (which the browser handles automatically) if redirected to Google for login
-                        console.log("Server responded \"" + stateData + "\" after previous problem \"" + previousState + "\"");
+                        $log.info("Server responded \"" + stateData + "\" after previous problem \"" + previousState + "\"");
```

Comment:
@jcabrerizo You need to add `$log` as the last parameter of the controller function (line 42)